include/rocksdb/cache.h

```diff
@@ -110,15 +115,21 @@ struct LRUCacheOptions {
                   bool _use_adaptive_mutex = kDefaultToAdaptiveMutex,
                   CacheMetadataChargePolicy _metadata_charge_policy =
                       kDefaultCacheMetadataChargePolicy,
-                  double _low_pri_pool_ratio = 0.5)
+                  double _low_pri_pool_ratio = kDefaultLowPriortyPoolRatio)
       : capacity(_capacity),
         num_shard_bits(_num_shard_bits),
         strict_capacity_limit(_strict_capacity_limit),
         high_pri_pool_ratio(_high_pri_pool_ratio),
-        low_pri_pool_ratio(_low_pri_pool_ratio),
+        low_pri_pool_ratio(_low_pri_pool_ratio == kDefaultLowPriortyPoolRatio
+                               ? 1.0 - _high_pri_pool_ratio
+                               : _low_pri_pool_ratio),
         memory_allocator(std::move(_memory_allocator)),
         use_adaptive_mutex(_use_adaptive_mutex),
-        metadata_charge_policy(_metadata_charge_policy) {}
+        metadata_charge_policy(_metadata_charge_policy) {
+    assert(high_pri_pool_ratio >= 0.0 && high_pri_pool_ratio <= 1.0);
+    assert(low_pri_pool_ratio >= 0.0 && low_pri_pool_ratio <= 1.0);
+    assert(high_pri_pool_ratio + low_pri_pool_ratio <= 1.0);
```

Comment:
I don't think these assertions are legit in the sense that they are not guaranteed hold when the object is created (the range checks are done in `NewLRUCache` IIRC)